src/control/system/membership.go

```diff
@@ -222,7 +222,8 @@ func (m *Membership) checkReqFaultDomain(req *JoinRequest) error {
 // AddOrReplace adds member to membership or replaces member if it exists.
 //
 // Note: this method updates state without checking if state transition is
-//       legal so use with caution.
+//
+//	legal so use with caution.
```

Comment:
I don't think adding the newline here is the right way to address this one. Should probably just condense the sentence to one line.

Reply:
I just ran the new formatter; didn't notice that it had done this. I think it was confused by the tab, so I just removed it.